packages/styles/components/_c.phone-number.scss

```diff
@@ -14,8 +14,8 @@
 
     display: flex;
     align-items: center;
-    color: $color-grey-900;
-    border: get-border("s") solid $color-grey-600;
+    color: $color-font-darker;
```

Comment:
Replace -darker to -primary

Reply:
@caqueste as seen together, this point will be fixed in a new PR on the `@next` version